plugins/woocommerce-blocks/assets/js/blocks/collection-filters/index.tsx

```diff
@@ -22,6 +31,124 @@ if ( isExperimentalBuild() ) {
 				/>
 			),
 		},
+		variations: [
+			{
+				name: 'woocommerce/collection-filters-variant',
+				title: __( 'Collection Filters (v)', 'woocommerce' ),
```

Comment:
I guess the `(v)` is for debugging purpose, right?

Reply:
Oops yes left this in by accident. Thanks for spotting.

Reply:
Resolved in https://github.com/woocommerce/woocommerce/pull/43216/commits/1692a4ef3381f442fb7dc7ab90977cf836753e8a